Resources/config/default_settings.yml

```diff
@@ -8,6 +8,12 @@ parameters:
     ezdemo.footer.latest_content.content_types: [article]
     # Number of blog_posts displayed per page in the blog view
     ezdemo.blog.blog_post_list.limit: 10
+    # Number of article displayed per page in the folder view
+    ezdemo.folder.folder_list.limit: 10
+    # ContentType identifiers we want to exclude from content list in folder view
+    ezdemo.folder.folder_content.excluded_content_types: [folder, image]
```

Comment:
I'd have used `folder_view.excluded_content_types` instead, as it is the list of "content types excluded from the folder view".